src/content/docs/mobile-monitoring/new-relic-mobile-android/install-configure/configure-android-gradle-android-studio.mdx

```diff
@@ -0,0 +1,139 @@
+---
+title:  Configure Android apps with Gradle and Android Studio
+tags:
+    - Mobile monitoring
+    - New Relic Mobile Android
+    - Install configure
+translate:
+    - jp
+    - kr
+metaDescription:
+redirects:
+    - /docs/mobile-monitoring/new-relic-mobile-android/install-configure/install-android-apps-gradle-android-studio/#4x-multidex
+    - /docs/mobile-monitoring/new-relic-mobile-android/install-configure/install-android-apps-gradle-android-studio/#configuration
+---
+
+Once you've [installed](/docs/mobile-monitoring/new-relic-mobile-android/get-started/introduction-new-relic-mobile-android/#installation) New Relic's agent for Android applications, configuring your mobile app can help to improve performance, stability, and user engagement of your app. To configure mobile monitoring with Gradle and Android Studio follow along below.
+
+## Configure with Gradle and Android Studio [#configuration]
+
+These procedures to configure your Android app with Gradle and Android Studio also appear on the **Get started** page in New Relic.
+
+1. Merge New Relic's mobile monitoring code in the **Gradle & Android Studio** tab to your `build.gradle` file.
+
+    <CollapserGroup>
+        <Collapser
```

Comment:
In the collapsers, consider adding context before the code samples. What's the difference between using app-level vs. project-level configuration, and when would I configure one over the other?